tasks/main.yml

```diff
@@ -1,9 +1,4 @@
 ---
-- include_vars: "{{ item }}"
-  with_first_found:
-    - "../vars/{{ ansible_os_family }}.yml"
```

Comment:
I moved the separate variables into vars/main.yml and the logic now resides within the tasks/debian.yml and tasks/redhat.yml. This allows to define versions and move to filebeat 5.